src/autoscaler/cf/config.go

```diff
@@ -55,5 +56,26 @@ func (conf *CFConfig) Validate() error {
 			return fmt.Errorf("Configuration error: client_id is empty")
 		}
 	}
+	if conf.UAAEndpoint != "" {
```

Comment:
why put uaa endpoint in config when it should be get from CC API /info?

Reply:
Removed